lib/node-token-validation/src/config/Configuration.ts

```diff
@@ -14,6 +14,10 @@ export type Configuration = {
 
 export type TokenValidationOptions = {
     authority?: string,
+    /**
+     * An array of URIs that are known to be valid. Used in B2C scenarios.
+     */
+    knownAuthorities?: Array<string>
     protocolMode?: ProtocolMode
```

Comment:
should there be commas at the end of these 2 lines?

Reply:
Yes! Fixed. Thank you